source/docs/uploaded.png

```diff
@@ -0,0 +1,103 @@
+<!--googleoff: all-->
```

Comment:
This file looks like HTML not PNG. Did you mean to upload it? It doesn't look referenced by the md